lib/email.js

```diff
@@ -20,12 +24,20 @@ bedrock.events.on('bedrock.test.configure', function() {
   require('./test.config');
 });
 
-var logger = bedrock.loggers.get('app');
-
 var api = {};
 module.exports = api;
 
+bedrock.events.on('bedrock.start', function(callback) {
+  brNotifications.register(api, function(err, results) {
```

Comment:
Do you mean something like bedrock-notifications firing off an "I'm ready" event that lets other modules subscribe to it? I like this way because other modules can potentially register or deregister themselves at different points in time as the application is running, whereas that might be a little tricky to handle with an event based system.